src/platform/EFR32/freertos_bluetooth.c

```diff
@@ -45,7 +45,11 @@ static volatile sl_bgapi_handler command_handler_func = NULL;
 #define BLUETOOTH_STACK_SIZE (1024)
 #endif
 static void BluetoothTask(void * p_arg);
-static TaskHandle_t BluetoothTaskHandle = NULL;
+volatile TaskHandle_t BluetoothTaskHandle = NULL;
```

Comment:
Why is this marked `volatile`?

Reply:
It appears this is so that read will work correctly in the `HeapMonitoring` while(1) loop.  However, that still presumes that pointer reads and writes are atomic, which is likely but not portable.

Reply:
Yeah this was for a direct use of the handle inside the monitoring task, however I could still revert it back to static and use `xTaskGetHandle` with the task name as to make sure that all task handle are queried the same way.

Reply:
Maybe, for the sake of future maintainers, it would be good to add comments and only have `volatile` if `HEAP_MONITORING` is set.

Also, it would be nice to keep the task handles static unless the build actually needs them to be externally accessible.

Reply:
It should be `std::atomic<TaskHandle_t>` then. volatile is not useful for this in C & C++.

Reply:
I see it was removed.